### PR TITLE
[5.2] Added verbosity level checking to output methods

### PR DESCRIPTION
**Pull request by timfeid**
Add verbosity level checking to info, line and comment output methods
This is an update to #10984

### CODE REVIEW

File: src/Illuminate/Console/Command.php [hunk 1]
```diff
@@ -319,68 +332,92 @@ public function table(array $headers, $rows, $style = 'default')
         $table->setHeaders($headers)->setRows($rows)->setStyle($style)->render();
     }
 
+     /**
+     * Convert the input level to the OutputInterface level
+     *
+     * @param  string|int  $level
+     * @return bool
+     */
+    public function convertVerbosity($level)
```
Comment (timfeid) [inline src/Illuminate/Console/Command.php]: Requesting suggestions for a better method name

File: src/Illuminate/Console/Command.php [hunk 2]
```diff
@@ -58,6 +58,19 @@ class Command extends SymfonyCommand
     protected $description;
 
     /**
+     * The mapping between human readable verbosity levels and Symfony's
```
Comment (GrahamCampbell) [inline src/Illuminate/Console/Command.php]: Please keep these summaries to one line.

**Comment by rkgrep** [comment]
I think some tests should be added here.

**Comment by taylorotwell** [comment]
I feel like I'm missing something. The verbosity level passed to line() is never used, nor is the convert function ever called anywhere.

**Comment by timfeid** [comment]
Uhhh.. this got reverted somehow. Let me fix this

File: src/Illuminate/Console/Command.php (hunk 1 above)
Comment (CupOfTea696) [inline src/Illuminate/Console/Command.php]: `getVerbosity` / `getVerbosityLevel` / `getRealVerbosityLevel` ?

Comment (CupOfTea696) [inline src/Illuminate/Console/Command.php]: Shouldn't this be a protected method as well?

File: src/Illuminate/Console/Command.php [hunk 3]
```diff
@@ -58,6 +58,18 @@ class Command extends SymfonyCommand
     protected $description;
 
     /**
+     * The mapping between human readable verbosity levels and Symfony's OutputInterface.
+     *
+     * @var int[]
```
Comment (CupOfTea696) [inline src/Illuminate/Console/Command.php]: Should be `@var array` I think.

**Comment by CupOfTea696** [comment]
I suggest adding a `verbosity` or `setVerbosity` method that would set a verbosity property that's used as a default for that command. Or even just the property. This way each command can set it's own default verbosity level rather than just hard coding it in the `convertVerbosity` method.

**Pull request merged**